meetings/2023-12-13.md

```diff
@@ -0,0 +1,127 @@
+* eP: W3C takes holiday break next week?
+* SC: Yes, I think 18th. Officially. 
+* eP: If they find it inconvenient to make a PR and send you data instead that'd be okay with you making the PR and they just approve or comment on that PR. I don't want to delay because they don't feel comfortable. 
+* SC: They're aware of the procedure. If there were issues they'd have mentioned or would let us know or just send me the data anyway. I don't expect any slowdown. They give us the data, we compute, we give to them, they approve and announce it. 
+
+
+### i18n and n11n of resource identifiers
+URL: https://github.com/solid/specification/pull/575
+
+* SC: All, please review.
+* SC: We're considering using IRIs instead of URIs, having mapping around.. gets a bit complicated. There's a number of specs involved. I reviewed, would be good to have more reviews. I'll follow up on my part. I think WT did a great job detailing what implementers need to do, but something like this needs deeper reviews, not just thumbs up to challenge each detail because things can be horribly wrong when ??? There's a parallel discussion on WebID CG about this topic. However we decide here will also influence how we use WebIDs. We need to have some implementation feedback on this. WT did a good job detailing what should happen, but not sure to what extent existing implementations will conform, and we don't have tests or a way to verify it. Hesitant to throw something like this into the spec and hoping things go smoothly. There's an alternative approach not to introduce this. If we use internationalized strings, we need to detail the conversion as for example WT ??? It comes down to how much interest people have. 
+* eP: Is there any changes regarding normalization? I see mention of an issue.. not sure if it's addressing in focused way. 
+* SC: There's a couple of issues across different repos touching on this topic. What WT did was gathering things that are impacted by this. WAC is another case. Kjetil covered it broadly. 
+* eP: I'll review and pick up again if needed.
+* SC: Be rough on it. Sometimes we let some features go through with very loose/non-strict reviews. This is one that needs a strong review. Challenge it to the point of "we don't need it", because other communities have avoided this possibility. I plan to compare it with approach to IRIs in [RDF Dataset Canonicalization](https://www.w3.org/TR/rdf-canon/)
+
+
+### Chair Election 2023 Retrospective
+
+* eP: For first meeting in January we can schedule retrospective on election process. Good opportunity for feedback/suggestions. 
+* eP: How about unlocking election procedure discussion and linking to that in case people have feedback?
+* SC: Kept it locked because I didn't want it to turn into a major discussion where people need to walk through threads and get into discussion. Going forward, I'd like to suggest we put this document into w3c-cg/solid repo and keep there, for future elections it can be copied.
```

Comment:
```suggestion
* SC: Kept it locked, because I didn't want it to turn into a major discussion where people need to walk through threads to get into the discussion. Going forward, I'd like to suggest we put this document into w3c-cg/solid repo and keep it there. For future elections, it can be copied.
```